site/source/_posts/2018/03/11/Dojo-2-rc1.md

```diff
@@ -5,33 +5,33 @@ authorId: dylan
 featured_image: featured.png
 ---
 
-We're pleased to announce that Dojo 2 release candidate 1 is now available!
+We're pleased to announce Dojo 2 release candidate 1 is now available!
 
-We first started brainstorming ideas for Dojo 2 back in 2010! While development of Dojo 2 did not quite take as long as Diablo 3 or Duke Nukem Forever, we followed a very deliberate journey along the way. Conceived initially as merely a modern clean-up for Dojo 1.x, with the introduction of ES6, TypeScript, web components, and an array of other features, we realized that we wanted to take a different path and create an approach to building modern apps that would not just be a slight clean-up of Dojo 1 or a clone of other current frameworks, but a productive and efficient development experience and front-end architecture.
+We first started brainstorming ideas for Dojo 2 back in 2010. While development of Dojo 2 did not take quite as long as Diablo 3 or Duke Nukem Forever, we followed a very deliberate journey. Conceived initially as a modern clean-up for Dojo 1.x, with the introduction of ES6, TypeScript, web components, and an array of other features, we realized we wanted to take a different path and create an approach to building modern apps that would not just be a slight clean-up of Dojo 1 or a clone of other current frameworks, but a productive and efficient development experience and front-end architecture.
```

Comment:
@dylans do we want to mention that it's taken 8 years to get here? doesn't sound very progressive. I feel that this first  paragraph should be very positive and should make it clear that `Dojo 2` is brand new, and exciting and game changing etc.. etc...

Reply:
See my other comment... I was thinking this would be more fun than negative... we took our time to get things right, etc.

Reply:
We could emphasize that releasing early before we felt right about the architecture would just create more work for our users, etc.?

Reply:
十年磨一剑(Ten years of grinding sword)

Reply:
@tomdye Please feel to suggest an alternative to the opening intro

Reply:
If @dylans wants to address the elephant in the room as he stated, then I guess it's fine.

Reply:
Otherwise:

> Dojo 2 is a completely new reactive framework written using ES6 and Typescript and aimed at modern app development. Dojo 2 provides a productive and efficient development experience and an opinionated front-end architecture. All aspects of modern app development have been considered and we hope to work with the community to continue adding targeted value. Out of the box, Dojo 2 delivers code completion, a build pipeline, internationalisation, accessible widgets and much more. We're excited to share our release candidate with you.